src/util.jl

```diff
@@ -12,7 +12,7 @@ Copied from [Trixi.jl](https://github.com/trixi-framework/Trixi.jl).
 readdir(examples_dir())
 ```
 """
-examples_dir() = pkgdir(DispersiveShallowWater, "examples")
+examples_dir() = pkgdir(DispersiveShallowWater, "examples")::String
```

Comment:
JET.jl complained that `pkgdir` may return `nothing` if the module is not known in the context, resulting in issues using `examples_dir()` in `joinpath`. However, we know that `DispersiveShallowWater` will be known here.